cocotb/binary.py

```diff
@@ -281,12 +281,30 @@ def _adjust_twos_comp(self, x):
             rv = x
         return rv
 
-    def get_value(self):
-        """Return the integer representation of the underlying vector."""
+    @property
+    def value(self):
+        """Integer access to the value. **deprecated**"""
+        return self.integer
+
+    @value.setter
+    def value(self, val):
+        self.integer = val
+
```

Comment:
Looking at the docs, this makes this appear before `integer`: https://cocotb--1497.org.readthedocs.build/en/1497/library_reference.html?highlight=binaryvalue#cocotb.binary.BinaryValue.value

Maybe move these lines down to line 301?